src/test/java/core/basesyntax/services/imp/ReportCreaterServiceImpTest.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.services.imp;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.services.ReportCreaterService;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class ReportCreaterServiceImpTest {
+
+    private final ReportCreaterService reportCreaterService = new ReportCreaterServiceImp();
+
+    @Test
+    public void create_correctReport_OK() {
+        Storage.storage.put("banana",50);
+        Storage.storage.put("apple",20);
+        String expected = "fruit,quantity"
+                + System.lineSeparator() + "banana,50"
+                + System.lineSeparator() + "apple,20"
+                + System.lineSeparator();
+        String actual = reportCreaterService.createReport();
+        Assert.assertEquals(expected,actual);
+    }
+
+    @Test
+    public void create_NullKeyAndValue_OK() {
```

Comment:
```suggestion
    public void create_nullKeyAndValue_OK() {
```
but I do not think we need this test case